buildpipeline/pipelinejobs.groovy

```diff
@@ -57,6 +57,7 @@ configurations.each { config ->
     // Add optional PR trigger for Outerloop test runs
     params.TestOuter = true
     pipeline.triggerPipelineOnGithubPRComment("Outerloop ${triggerName}", params)
+    pipeline.triggerPipelinePeriodically('@daily', params)
```

Comment:
What value does this give us over the results of our daily scheduled official runs (2)?

Reply:
Where are those defined? I didn't see any Outerloop daily runs, but I probably wasn't looking in the right place.

Reply:
The schedule is defined (manually, unfortunately) in the VSTS schedule, the builds it ends up hydrating and using are in this directory.

Reply:
Ah, gotcha. Well if dotnet/core-eng#1901 allows us to point badges at the trusted VSTS runs, then we don't need to add them here also.